Template/Tag/OneTrustTag.web.js

```diff
@@ -0,0 +1,16 @@
+(function () {
+  return function (parameters, TagManager) {
+    this.fire = function () {
+      const domain = parameters.get('domain');
+
+      (function (d, s) {
+        const t = d.getElementsByTagName(s)[0]; const
+          e = d.createElement(s);
```

Comment:
It doesn't need to be changed, but I think it's easier to read on separate lines, especially since `const` should be on the same line as the variable definition.
```suggestion
        const t = d.getElementsByTagName(s)[0];
        const e = d.createElement(s);
```